src/ui/public/index_patterns/components/index_pattern_select.js

```diff
@@ -24,15 +24,16 @@ import chrome from 'ui/chrome';
 
 import { EuiComboBox } from '@elastic/eui';
 
-const getIndexPatterns = async (search) => {
+const getIndexPatterns = async (search, fields, filterFields) => {
```

Comment:
@aaronjcaldwell Can you make `fields` and `filterFields` optional? `IndexPatternSelect` is used in other parts of kibana and we should not break the interface. I would also recommend moving these changes into a seperate PR directly against Kibana so we can merge these sooner to avoid as many merge conflicts as possible